project-reports/2023/2023-Q1-Hyperledger-AnonCreds.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: 2023 Q1 Hyperledger AnonCreds
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# 2023 Q1 Hyperledger AnonCreds
+
+Created by Stephen Curran.
+
+## Implementation Repositories
+
+- [anoncreds-rs] - Rust implementation of AnonCreds
+
+[anoncreds-rs]: https://github.com/hyperledger/anoncreds-rs
+
+### Specifications
+
+- [AnonCreds Specification] published from the repository [anoncreds-spec]
+- [AnonCreds Methods Registry] published from the repository [anoncreds-methods-registry]
+
+[anoncreds-spec]: https://github.com/hyperledger/anoncreds-spec
+[AnonCreds Specification]: https://hyperledger.github.io/anoncreds-spec/
+[anoncreds-methods-registry]: https://github.com/hyperledger/anoncreds-methods-registry
+[AnonCreds Methods Registry]: https://hyperledger.github.io/anoncreds-methods-registry
+
+### Web and Wiki Pages
+
+- [Hyperledger AnonCreds Website Home Page]
+- [Hyperledger AnonCreds Wiki Home Page]
+
+[Hyperledger AnonCreds Website Home Page]: https://www.hyperledger.org/use/anoncreds
+[Hyperledger AnonCreds Wiki Home Page]: https://wiki.hyperledger.org/display/ANONCREDS/
+
+## Project Health
+
+This is the first report from the Hyperledger AnonCreds project. The project was
+announced on November 15, 2022 with this [Hyperledger blog post announcement].
```

Comment:
Seems like there's a redundant "[Hyperledger blog post announcement]" here. Perhaps you should remove the line below, and instead wrap the link around the above text.